gui/qt/contact_list.py

```diff
@@ -33,7 +33,7 @@
 from PyQt5.QtWidgets import (
     QAbstractItemView, QFileDialog, QMenu, QTreeWidgetItem)
 from .util import MyTreeWidget
-
+import os, json
```

Comment:
Please follow [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for the import order:

> Imports should be grouped in the following order:
>
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports

So this should be moved up to `import webbrowser`.

Reply:
done